src/App.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import { BrowserRouter as Router, Route } from 'react-router-dom';
-import { Join } from './components/Join';
-import { Chat } from './components/Chat';
+import { SignIn } from './pages/signIn';
+import { SignUp } from './pages/signUp';
 
 const App = () => (
   <Router>
-    <Route path="/" exact component={Join} />
-    <Route path="/chat" component={Chat} />
+    <Route path="/" exact component={SignIn} />
+    <Route path="/signup" component={SignUp} />
   </Router>
 );
```

Comment:
I would move the entire `Router` to a separate file under `src/routes/AppRouter.js`. As more and more routes being added to the web app, it will clutter `App.js` eventually. See [unifree](https://github.com/JiejayLan/unifreefrontend/blob/develop/src/routes/AppRouter.js).

Also, I would reserve `path = "/"` to the home page and move `SignIn` component to `path = "/signin"`.